src/main/java/com/example/hellotalk/exception/AppExceptionHandler.java

```diff
@@ -18,10 +18,10 @@ public class AppExceptionHandler extends ResponseEntityExceptionHandler {
     public static final String COMMENT_NOT_FOUND_EXCEPTION = "NO COMMENT FOUND WITH THIS ID";
     public static final String MOMENT_ALREADY_LIKED_EXCEPTION = "MOMENT ALREADY LIKED";
     public static final String MOMENT_NOT_YET_LIKED_EXCEPTION = "MOMENT NOT YET LIKED";
-    public static final String FOLLOWSHIP_ALREADY_EXISTS_EXCEPTION = "FOLLOWSHIP DELETED AS IT ALREADY EXISTED";
-    public static final String FOLLOWSHIP_NOT_CREATED_USER_CANT_FOLLOW_THEMSELF = "USER TRYING TO FOLLOW THEMSELF. SENDER AND RECEIVER USERS CAN'T BE THE SAME";
```

Comment:
Not related to the testcontainers update. Only the BasePostgresConfig class and the deletion for the test related yml files and unused classes.